install.sh

```diff
@@ -4,15 +4,21 @@ set -e
 
 # This script simply builds the docker image and copies the script
 # over to $HOME/bin. 
-
 # This must be run in the root directory of the dockerized-whistle
 # repository, since that is where the Dockerfile is kept.
 
-docker build --build-arg DOCKER_USER=$USER -t torstees/whistle .
+# if the user has Macbook with amd64, select compatible platform
+PLATFORM=""
+if PLATFORM=$(arch); then
+    if PLATFORM="arm64"; then
```

Comment:
This if does work correctly for your mac, but on linux/WSL, it leaves platform with the output from line 12 which messes up the docker call on line 18. My suggestion is to assign/test something other than PLATFORM with regards to the arch call. So, maybe if ARCH=$(arch); then...if ARCH="arm64"; then...PLATFORM=blah.   This would leave PLATFORM empty for non arm64 hardware including WSL.